### DISCUSSION TITLE
Self-hosted WC Relay

### DISCUSSION THREAD

**Discussion by renestefancic** [WalletConnect/walletconnect-monorepo#4628] [Developer Support]
Hi,

There hasn't been an update to self-hosting a WC Relay in a long time. We are highly interested in this, can we get an update from the team on an ETA please?

**Comment by glitch-txs** [comment]
We are working on decentralizing the Network, but there's not a fixed ETA ATM. You can read more about it [here](https://walletconnect.com/blog/the-next-phase-of-the-walletconnect-protocol).
